src/symon/index.ts

```diff
@@ -325,8 +325,11 @@ class SymonClient {
 
   private async fetchProbes() {
     log.debug('Getting probes from symon')
+    const TIMEOUT = 30_000
+
     return this.httpClient
       .get<{ data: Probe[] }>(`/${this.monikaId}/probes`, {
+        timeout: TIMEOUT,
```

Comment:
Just a simple request, could you make it override-able? So that we can test it in the Symon using a flag to know which sweet spot for each location

Reply:
This PR is a workaround. I'm currently working on the long term solution.